src/token/IRift.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.11;
+
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC20/IERC20Upgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/access/IAccessControlUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/draft-IERC20PermitUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/governance/utils/IVotesUpgradeable.sol";
+
+interface IRift is IERC20Upgradeable, IERC20PermitUpgradeable, IAccessControlUpgradeable, IVotesUpgradeable {
+    // roles
+    function MINTER_ROLE() external view returns (bytes32);
+
+    function DEFAULT_ADMIN_ROLE() external view returns (bytes32);
```

Comment:
It's not, I guess because it's a constant?
Note: we don't necessarily need to include it - I did because I switched to using the interface in the tests, but can revert to using the contract directly.

Reply:
got it, let's keep it then